styles/simple_print.css

```
@@ -21,7 +24,7 @@
  * themes include an image or border
  */
 
-@page:first {
+@page: first {

```

Is this space intentional? Comparing this to what's in `master` no space has been added.
```suggestion
@page:first {

```